pinc/theme.inc

```diff
@@ -184,7 +184,7 @@ function html_logobar() {
             $id = 'testbar';
         }
         if($alert_message)
-            $messages[] = $alert_message;
+            $messages[] = html_safe($alert_message);
```

Comment:
We allow HTML tags (like `<a>`) in `$alert_message` so we assume(!) that the admins who set the alert message make it HTML safe and should not escape it here.

Reply:
I only noticed because "<<ALERT_MESSAGE>>" was in the default and it wasn't encoded and the validator complained, but that makes sense. I can change the default to &lt;&lt;...but probably overkill.

Reply:
That value is replaced by `configure` as part of set-up. If it's still that value in your sandbox you might want to reconfigure your sandbox on TEST with:
```
SETUP/configure ~cpeel/bin/TEST_setup.sh .
```

If you're testing locally, check the updated `SETUP/configuration.sh` script.